pkg/data/management/role_data.go

```diff
@@ -452,6 +452,7 @@ func addUserRules(role *roleBuilder) *roleBuilder {
 	role.
 		addRule().apiGroups("").resources("secrets").verbs("create").
 		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("get", "list", "watch").
+		addRule().apiGroups("catalog.cattle.io").resources("operations").verbs("get", "list", "watch").
```

Comment:
Won't this allow a standard user to see any operation?

Reply:
Yes... I can limit it to `fleet-default` namespace, but you are still able to other user's operation.
Unless we go through the route that only who create operation can see it and view logs, but that requires more efforts to implement.

Reply:
We spoke offline, but yes, the permissions have to be specific to the user and object.